gen/definitions/profile_parcels/service_lan_vpn.yaml

```diff
@@ -0,0 +1,423 @@
+    # ====== UX2.0 Route Policy Not Supported By Provider ======
+    - model_name: refId
+      tf_name: route_policy_id
+      data_path: [routePolicy]
+      id: true
+      example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+      # test_value: 
+    - model_name: prefixList
+      tf_name: prefix_lists
+      exclude_test: true
+      attributes:
+      - model_name: address
+        tf_name: network_address
+        data_path: [prefix]
+        id: true
+        example: 1.2.3.4
+      - model_name: mask
+        tf_name: subnet_mask
+        data_path: [prefix]
+        example: 255.255.255.255
+      - model_name: aggregateOnly
+        example: false
+      - model_name: region
+        example: core-and-access
+  - model_name: ompAdvertiseIpv6
+    tf_name: advertise_omp_ipv6s
+    exclude_test: true
+    attributes:
+    - model_name: ompProtocol
+      tf_name: protocol
+      example: OSPF
+    # ====== UX2.0 Route Policy Not Supported By Provider ======
+    - model_name: refId
+      tf_name: route_policy_id
+      data_path: [routePolicy]
+      id: true
+      example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+      # test_value: 
+    - model_name: protocolSubType
+      example: External
+    - model_name: prefixList
+      tf_name: prefix_lists
+      exclude_test: true
+      attributes:
+      - model_name: prefix
+        id: true
+        example: 2001:db8:a::/64
+      - model_name: aggregateOnly
+        example: false
+  - model_name: ipv4Route
+    tf_name: ipv4_static_routes
+    attributes:
+    - model_name: ipAddress
+      tf_name: network_address
+      data_path: [prefix]
+      id: true
+      example: 1.2.3.4
+    - model_name: subnetMask
+      data_path: [prefix]
+      example: 0.0.0.0
+    - model_name: nextHop
+      tf_name: next_hops
+      data_path: [oneOfIpRoute, nextHopContainer]
+      attributes:
+      - model_name: address
+        id: true
+        example: 1.2.3.4
+      - model_name: distance
+        tf_name: administrative_distance
+        example: 1
+    - model_name: nextHopWithTracker
+      tf_name: next_hop_with_trackers
+      data_path: [oneOfIpRoute, nextHopContainer]
+      exclude_test: true
+      attributes:
+      - model_name: address
+        example: 1.2.3.4
+      - model_name: distance
+        tf_name: administrative_distance
+        example: 1
+      # ====== UX2.0 Tracker and Trackerobject Not Supported By Provider ======
+      - model_name: refId
+        tf_name: tracker_id
+        data_path: [tracker]
+        example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+       # test_value: 
+    - model_name: null0
+      data_path: [oneOfIpRoute]
+      exclude_test: true
+      example: false
+    - model_name: dhcp
+      tf_name: gateway_dhcp
+      data_path: [oneOfIpRoute]
+      exclude_test: true
+      example: false
+    - model_name: vpn
+      data_path: [oneOfIpRoute]
+      exclude_test: true
+      example: false
+  - model_name: ipv6Route
+    tf_name: ipv6_routes
```

Comment:
For ipv4 we call it `ipv4_static_routes`. Let's maybe check how we name this in other places and make it consistent.